### DISCUSSION TITLE
Search/Filter Within A Playlist & Album

### DISCUSSION THREAD

**Discussion by Menghini** [audioling/audioling#4] [Ideas]
I am an avid user of Sonixd. I have been following the development of Feishin as it slower gained features that came close to the features that were in Sonixd. However, I was never able to make the switch over to Feishin as I could not search within a playlist or album. In almost every single instance that I open up a music player, I am searching for a song, and I simply could not do so in Feishin without searching the entire library at once. It surprised me that Feshin was being rewritten when it still didn't have all the features of Sonixd!

Nevertheless, I ask that Audioling has the implementation of searching/filtering within an album and playlist, without having to search the entire library of songs. As mentioned, this is the strictly the only reason why I am still using Sonixd.
 
This request is the same as [issue #746 in Feishin](https://github.com/jeffvli/feishin/issues/746)

**Comment by jeffvli** [comment]
Honestly I'm not sure why I never got around to implementing this in Feishin. I will definitely add this here.

**Reply by Menghini** [reply]
Much, much appreciated! I look forward to the new program! Thanks for your great work!
